Explorer/Assets/DCL/AvatarRendering/Wearables/Helpers/WearableComponentsUtils.cs

```diff
@@ -63,11 +77,41 @@ public static GetWearablesByPointersIntention CreateGetWearablesByPointersIntent
                 return null;
             }
 
+            if (attachment.ManifestResult?.Asset == null)
```

Comment:
Could you extract this block into smaller methods so its more descriptive?

Reply:
Done!